examples/bri-3/prisma/migrations/20230104084453_add_document_table/migration.sql

```diff
@@ -0,0 +1,23 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[documentId]` on the table `CCSMAnchorHash` will be added. If there are existing duplicate values, this will fail.
+  - Added the required column `documentId` to the `CCSMAnchorHash` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "CCSMAnchorHash" ADD COLUMN     "documentId" INTEGER NOT NULL;
+
+-- CreateTable
+CREATE TABLE "Document" (
```

Comment:
This is just a discussion point: What does this entity represent? If the BPI serves as the state synchronizer under zero knowledge, we would need to avoid storing customer data in plain text. Or allow it only if the workgroup participants decide to do it. But it makes much more sense to me that the data resides in respective systems of record while sync proofs reside in the BPI. Could you please post a link to the document where this entity design was outlined? I might have missed it completely, sorry.

Reply:
This design for the document is outdated. I will change it. As clarified by Andreas in last week's call and his above comment, we will not hash or store the document directly. We will hash and store the `stateObjectStorage` (which includes the document) parameter of `BPIAccount`, i.e., we are hashing and syncing the BPIAccount state.